.pre-commit-config.yaml

```diff
@@ -1,14 +1,28 @@
 repos:
   - repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.56.0
+    rev: v1.58.0
     hooks:
       - id: terraform_fmt
       - id: terraform_validate
       - id: terraform_docs
         args:
           - '--args=--lockfile=false'
+      - id: terraform_tflint
```

Comment:
`terraform_tflint` should not be here because this module contains incorrect code in `modules/_templates` which is used by a script `update_groups.sh`.

In general, I think we can have the release process as in all other modules.